ingest/usecase/plugins/cyclicarb/README.md

```diff
@@ -1,14 +1,65 @@
 # Cyclic Arb Plugin
 
+## Overview
 
-- What do we want?
+The Cyclic Arbitrage Plugin is a component designed to identify and execute profitable trading opportunities across osmosis liquidity pools. It automatically detects price inefficiencies in the marketplace and performs trades to capture arbitrage profit.
 
-- Concurrency of processing pools
+## How It Works
 
-- Lock on a single signer as to avoid nonce issues
+Cyclic arbitrage is based on a simple principle:
 
-- Signer queue so that we do not get bottlecked on one signer (optional)
+- Start with an amount of Token A
```

Comment:
The way we determine this amount is by binary search btw

We set a heuristic of let's say 5 and then binary search simulate what would yield the most $

Reply:
https://github.com/czarcas7ic/sqs/blob/eb4a7459d7411a8b452d94f53dda6daa621e7285/ingest/usecase/plugins/cyclicarb/README.md#L27-L35

Thats explained here right?